lib/actionmailer-text/html_to_plain_text.rb

```diff
@@ -32,12 +32,12 @@ def self.convert_to_text(html, line_length = 65, _from_charset = 'UTF-8')
         txt.gsub!(/<img.+?alt='([^\']*)\'[^>]*\/>/i, '\1')
 
         # links
-        txt.gsub!(/<a.+?href=\"([^\"]*)\"[^>]*>(.+?)<\/a>/mi) do |_s|
-          Regexp.last_match[2].strip + ' ( ' + Regexp.last_match[1].strip + ' )'
+        txt.gsub!(/<a.+?href=\"([^\"]*)\"[^>]*>(.*?)<\/a>/mi) do |_s|
+          Regexp.last_match[2].strip + ' ( ' + Regexp.last_match[1].strip + ' )' unless Regexp.last_match[2].strip.empty?
         end
 
-        txt.gsub!(/<a.+?href='([^\']*)\'[^>]*>(.+?)<\/a>/mi) do |_s|
-          Regexp.last_match[2].strip + ' ( ' + Regexp.last_match[1].strip + ' )'
+        txt.gsub!(/<a.+?href='([^\']*)\'[^>]*>(.*?)<\/a>/mi) do |_s|
+          Regexp.last_match[2].strip + ' ( ' + Regexp.last_match[1].strip + ' )' unless Regexp.last_match[2].strip.empty?
```

Comment:
Good. Lets avoid calling `strip` twice, assign `Regexp.last_match...` to a variable and then use them, maybe:

``` ruby
matches = Regexp.last_match.map(&:strip)
matches[2]
```